concepticondata/conceptlists/Savelyev-2020-254.tsv

```diff
@@ -66,7 +66,7 @@ Savelyev-2020-254-64	64	fat (n.)	323	FAT (ORGANIC SUBSTANCE)
 Savelyev-2020-254-65	65	father (n.)	1217	FATHER
 Savelyev-2020-254-66	66	fear (v.)	1419	FEAR (BE AFRAID)
 Savelyev-2020-254-67	67	feather (n.)	1201	FEATHER
-Savelyev-2020-254-68	68	female (of an animal) (n.)	2262	FEMALE (OF ANIMAL)
+Savelyev-2020-254-68	68	female (of an animal) (n.)		
```

Comment:
This list is derived from the list of Key 1310 and the Leipzig Jakarta list, etc., so the "female of animal" should be mapped to 2262.

Reply:
As far as I can see Savelyev is based on the Swadesh list of 200 items and the Leipzig-Jakarta list but neither contains a similar concept. But the main reason for unmapping this gloss was the ontological category of FEMALE (OF ANIMAL). It's marked as `Property `and is narrower than FEMALE which is used to denote the attribute by the same token. On contrary, the mapping of `female (adj)` in Key-2016-1310 to the mapping to FEMALE (OF ANIMAL) is fully justified.

Reply:
so you argue that the distinction is that this is a noun, while the concepticon entry is an adjective, such as "Weibchen vs. Männchen" in German is the noun translation of "female vs. male of animal", while "weiblich vs. männlich" would be used as adjective readings. Since Leipzig-Jakarta is derived from Key (or better: WOLD = Haspelmath-2008-1460), I think they got the inspiration of the concept from there. But it is not entirely clear. I am fine with un-mapping now.

Reply:
In Haspelmath-2008-1460 the semantic category of the concept female (2) mapped to FEMALE (OF ANIMALS) is also the adjective ([see here](https://wold.clld.org/meaning/3-13#2/24.3/-4.8)). I can see the connection but I'm hesitant to map the substantive gloss here to the clearly adjectival concept set.

Reply:
Yes, I agree. I think it may be useful to have a look at the Savelyev publication to see if they mention that they used other concept lists than Swadesh and LJ. I mean, obviously, they did. This may even ben mentioned in our NOTE to the list.